pkg/api/service/util_test.go

```diff
@@ -129,6 +129,43 @@ func TestAllowAll(t *testing.T) {
 	checkAllowAll(true, "192.168.0.1/32", "0.0.0.0/0")
 }
 
+func TestExternallyAccessible(t *testing.T) {
+	checkExternallyAccessible := func(expect bool, service *api.Service) {
+		res := ExternallyAccessible(service)
+		if res != expect {
+			t.Errorf("Expected ExternallyAccessible = %v, got %v", expect, res)
+		}
+	}
```

Comment:
call `t.Helper()` as the first line of `checkExternallyAccessible`, so that if `t.Errorf()` gets called, it will report that the failure occurred on the line number of the call to `checkExternallyAccessible()`, rather than always claiming the error occurred on line 136.

EDIT: oh, I see, you copied this pattern from other tests... maybe consider a follow-up PR to fix all of them...